cli/ballerina-cli/src/main/resources/cli-help/ballerina-bindgen.help

```diff
@@ -13,32 +13,34 @@ DESCRIPTION
        Generate Ballerina bridge code for Java APIs.
 
        Ballerina bindings could be generated for Java classes residing inside Java libraries
-       or for standard Java classes. The Java classes will be mapped to the Ballerina
-       classes providing a seamless Java interoperability developer experience to Ballerina users.
+       or standard Java classes. The Java classes will be mapped onto Ballerina
+       objects making the developer experience of Ballerina Java interoperability seamless.
 
-       In addition to the user-specified Java classes, partial implementations of directly
-       -dependent Java classes will also be generated by the tool. By default, the bindings for
-       each Java package will be mapped to a separate Ballerina module.
+       The directly-dependent Java classes and other required resources will be automatically
+       generated apart from the specified Java classes.
 
 
 OPTIONS
        (-cp|--classpath) <classpath>...
-           One or more comma-delimited classpaths for obtaining the jar files required for
+           One or more comma-delimited classpaths for obtaining the JAR files are required for
```

Comment:
This describes what the `classpath` option is. Therefore, we should remove the additional word "are" since it does not mean the `classpath` option is mandatory. It just means you can add it if it is required for generating the bindings.
```suggestion
           One or more comma-delimited classpaths for obtaining the JAR files required for
```